astropy/coordinates/sky_coordinate.py

```diff
@@ -1225,6 +1225,92 @@ def from_pixel(cls, xp, yp, wcs, origin=0, mode='all'):
         """
         return pixel_to_skycoord(xp, yp, wcs=wcs, origin=origin, mode=mode, cls=cls)
 
+    def radial_velocity_correction(self, kind='barycentric', obstime=None,
+                                   location=None):
+        """
+        Compute the correction required to convert a radial velocity at a given
+        time and place to a barycentric or heliocentric velocity.
+
+        Parameters
+        ----------
+        kind : str
```

Comment:
I hate string arguments (just needed to get that out). 

But seriously, is there anything we can do to not have to ask people to type and spell `'barycentric'`? I don't have any other immediate ideas...so probably the answer is no!